src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -265,19 +267,17 @@ public static void openConsole(File file) throws IOException {
         if (file == null) {
             return;
         }
-
-        String absolutePath = file.getAbsolutePath();
-        absolutePath = absolutePath.substring(0, absolutePath.lastIndexOf(File.separator) + 1);
+        String absolutePath = file.toPath().toAbsolutePath().getParent().toString();
```

Comment:
Just out of interest: what is the difference between `file.toPath().toAbsolutePath()` and `file.getAbsolutePath()`?